.github/workflows/institution-e2e-tests.yml

```diff
@@ -58,6 +58,8 @@ jobs:
           echo WEB_UI_CLIENT_SECRET=${{ secrets.WEB_UI_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
           echo E2E_INSTITUTION_PASSWORD=${{ secrets.E2E_INSTITUTION_PASSWORD }} >> ${ENV_FILE_PATH}
           echo SUPER_ADMIN_PASSWORD=${{ secrets.SUPER_ADMIN_PASSWORD }} >> ${ENV_FILE_PATH}
+          echo AGGREGATOR_ADMIN_USERNAME=${{ secrets.AGGREGATOR_ADMIN_USERNAME }} >> ${ENV_FILE_PATH}
```

Comment:
Usually we put the non-sensitive stuff into a variable then you don't need to do this.